README.md

```diff
@@ -9,79 +9,76 @@
 
-- Check if a tag exists.
-- Check the changelog format.
-- Compare two changelog files.
-- Add a new tag.
-- Add a new unreleased entry.
+```txt
+- An unreleased change
 
-## Installation
+v0.2.0
+- A change
 
-You can use Changelogtxt-parser in two ways:
-
-1. Install in your project:
-
-<div class="termy">
-
-```console
-uv add git+https://github.com/geopozo/changelogtxt-parser
+v0.1.0
+- A change
+- Another change
 ```
 
-</div>
+### How to Install
 
-After installation, verify that it’s working by running:
+```shell
+$ uv add git+https://github.com/geopozo/changelogtxt-parser
+# or
+$ pip install git+https://github.com/geopozo/changelogtxt-parser
+```
 
-<div class="termy">
+## Python API
 
-```console
-uv run changelogtxt --help
+```python
+import changelogtxt
+x = changelogtxt.load(filename)
+# ejemplo objeto
+changelogtxt.dump(object)
 ```
 
-</div>
+## CLI Examples
 
-2. From source using uvx (recommended for latest version):
+```shell
+# lint
+$ changelogtxt check-format
 
-<div class="termy">
+# verify version exists
+$ changelogtxt get-tag v1.0.1
 
-```console
-uvx --from git+https://github.com/geopozo/changelogtxt-parser changelogtxt --help
-```
+# add new change or version
+$ changelogtxt update -t "v1.0.2" -m "Change"
 
-</div>
+# compare two git ref files
+$ changelogtxt summarize-news <origin> <target>
+```
 
-## Using Changelogtxt-parser
+## Basic action
 
-<div class="termy">
+```yaml
+- name: Check changelog
+  uses: geopozo/changelogtxt-parser@main
+  with:
+    # Python version to use (default: 3.12)
+    python-version: ""
 
-```console
-usage: changelogtxt [-h] [--logistro-human]
-[--logistro-structured] [--logistro-level LOG]
-{check-tag,check-format,summarize-news,update} ...
+    # Path to the changelog file (default: searches ./CHANGELOG.txt)
+    file-path: ""
 
-changelogtxt helps you manage your changelog file.
-changelogtxt COMMAND --help for information about commands.
+    # Whether to validate the changelog format (default: "true")
+    check-format: "true"
 
-positional arguments:
-  {check-tag,check-format,summarize-news,update}
-    check-tag           Checks if a tag in the changelog
-                        matches the specified tag.
-    check-format        Check changelog format.
-    summarize-news      Compare source file with target file.
-    update              Creates a new version entry if it
-                        doesn't exist.
+    # Tag to verify. Use "from-push" to get the tag from the latest push
+    get-tag: "v1.0.0"
 
-options:
-  -h, --help            show this help message and exit
-  --logistro-human      Format the logs for humans
-  --logistro-structured
-                        Format the logs as JSON
-  --logistro-level LOG  Set the logging level (no default,
-                        fallback to system default)
+    # Compare changelogs from current to provided branch
+    summarize-news: '["<file_path>", "<target_branch>"]'
```

Comment:
is `file_path` relative to `working_dir` or what? probably.

`target_branch` can be any ref?

`<target_branch>` can be branch name, commit has, tag, etc.